app/client/stylesheets/_login.scss

```diff
@@ -9,10 +9,28 @@
 #login {
   height: 100%;
   width: 100%;
-  background-color: $login-bg-color;
-  background: $login-bg;
-  background-size: cover;
-  overflow: hidden;
+
+  > img.background {
+    height: 100%;
+    width: 101%;
+    overflow: hidden;
+    z-index: -1;
```

Comment:
This use of z-index can be avoided by just moving `#particles-js` div below the `#login` div.

Reply:
oh i actually didnt need that at all i found. ill remove it